shim/opentelemetry-opencensus-shim/pyproject.toml

```diff
@@ -32,12 +32,6 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-test = [
-  "opentelemetry-test-utils == 0.45b0.dev",
```

Comment:
Is this not needed anymore?

Reply:
These test dependencies are being added now in the test-requirements files.

Reply:
Then it is not needed. I checked and there is no instance of `util` in `opentelemetry-python/shim/opentelemetry-opencensus-shim/tests`.